### DISCUSSION TITLE
突然有一篇文献的引用出现了名

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#140] [Q&A]
你好，文献在文中引用不是只有姓吗，突然有一篇出现了名：

> @article{RN226,
   author = {Zhang, Shengnan and Tian, Yimei and Guo, Hao and Liu, Ran and He, Nan and Li, Zhuang and Zhao, Weigao},
   title = {Study on the occurrence of typical heavy metals in drinking water and corrosion scales in a large community in northern China},
   journal = {Chemosphere},
   volume = {290},
   note = {Times Cited: 0
zhao, weigao/0000-0002-4955-1626
0
22
1879-1298},
   abstract = {Excessive heavy metal content in drinking water could lead to red water and acute and chronic diseases. A field study in combination with batch experiments using pipe scales of drinking water distribution systems (DWDS) in the study area, was used to determine the content distribution and migration of As, Cd, Cr, Mn, Pb, and V in DWDS. In the field study, As, Cd, Cr, Pb, and V contents in pipe scales and drinking water were extremely low and did not exceed the Chinese drinking water standards. However, Mn concentrations at the end of the DWDS with aged and corrosive pipes were relatively high, which presented a risk of excessive release. The batch experiment showed that As in pipe scales would not be released into water under static immersion conditions; however, pipe scales would release excessive Cd, Cr, Mn, Pb, and V in the initial reaction stage, and the heavy metal contents released by tubercle scales in the initial release stage were at least twice as much as those released by loose scales. The mass percentage of four metals (excluding Cd and Pb) released from pipe scales was extremely low. The field study and batch experiment data both suggested a strong correlation between Cr and V released into the water, indicating a synergistic effect. There were differences in heavy metals released in the field research and the batch experiment. The amount of Cd, Cr, Pb, and V released were not consistent with its proportion in pipe scales. As release did not occur under static conditions, but may be promoted by the water flow in the actual network. The effect of water flow on heavy metal release in DWDS should be considered.},
   ISSN = {0045-6535},
   DOI = {10.1016/j.chemosphere.2021.133145},
   url = {<Go to ISI>://WOS:000753975700003
https://www.sciencedirect.com/science/article/pii/S0045653521036171?via%3Dihub},
   year = {2022},
   type = {Journal Article}
}

![image](https://user-images.githubusercontent.com/45897471/169298027-e2e8e8c4-9a25-45f4-9a25-1ad0b37df967.png)
这是啥情况啊？
谢谢

**Comment by tanukihee** [comment]
可能出现了同姓的，建议检查规范和文献列表

**Reply by earthmoon** [reply]
> 可能出现了同姓的，建议检查规范和文献列表

明白了，谢谢！

**Reply by zepinglee** [reply]
这是姓名的消歧义方式。

另外建议标题不要含有“突然”、“昨天还好好的”这类词或短语。

**Reply by earthmoon** [reply]
好的，谢谢。
